app/main/routes.py

```diff
@@ -205,18 +205,25 @@ def search():
     )
     page = int(request.args.get("page", 1))
     filters = {"query": query}
+    sorting_orders = build_sorting_orders(request.args)
 
     if query:
-        fuzzy_search_query = build_fuzzy_search_query(query)
+        query = build_fuzzy_search_query(
+            query,
+            sorting_orders=sorting_orders,
+        )
+        # added a filter for transferring body - for standard user to return only matching rows
         ayr_user = AYRUser(session.get("user_groups"))
         if ayr_user.is_standard_user:
-            fuzzy_search_query = fuzzy_search_query.where(
-                Body.Name == ayr_user.transferring_body.Name
-            )
-        search_results = fuzzy_search_query.paginate(
-            page=page, per_page=per_page
-        )
-        num_records_found = search_results.total
+            query = query.where(Body.Name == ayr_user.transferring_body.Name)
+
+        search_results = query.paginate(page=page, per_page=per_page)
+
+        total_records = query.count()
```

Comment:
Isnt the only option here for query to be a positive integer or 0 so we should be fine by just doing `num_record_found = query.count()`. Just noticed this here, will make same comment on the browse PR.

Reply:
ok